### ISSUE TITLE
Create a CONTRIBUTING.md

### ISSUE THREAD

**Issue by wanderingstan** [OriginProtocol/origin-js#177]
Note sure if final content lives in repos, on dev landing page, or where, but we should have this content somewhere. 

Examples: 

- https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/CONTRIBUTING.md
- https://reactjs.org/docs/how-to-contribute.html
- https://github.com/angular/angular/blob/master/CONTRIBUTING.md

Thanks to @gaboesquivel for additional suggestions from discord.

**Comment by joshfraser** [comment]
I've been thinking we would use http://docs.originprotocol.com/#contributing as the canonical guide for how to contribute versus trying to maintain a separate file in each repo, especially given how quickly stuff changes around here.

Maybe each repo can have a CONTRIBUTING.md file with just a link to that section of the docs?

**Comment by wanderingstan** [comment]
Yeah, having CONTRIBUTING.md with the link would cover the bases for people that look for that.

**Comment by gaboesquivel** [comment]
Consider adopting Contributor Covenant code of conduct and link to it in the contributing docs https://www.contributor-covenant.org/version/1/4/code-of-conduct.html.  This code of conduct has already been adopted by over 40,000 open source projects. https://www.contributor-covenant.org/#adopters-of-the-contributor-covenant
